themes/wdfn_theme/layouts/index.html

```diff
@@ -0,0 +1,12 @@
+{{ define "main" }}
+  <div class="grid-row grid-gap-md">
+    {{ range $key, $value := .Paginator.Pages }}
+      {{ if ge $value.Date.Unix 100 }}
+        <div class="mobile-lg:grid-col-6 tablet:grid-col-4 desktop:grid-col-3">
```

Comment:
I believe we could also handle the grid with mixins. What are your thoughts on using that rather than the utility classes?

Reply:
Yes I think they can. I wouldn't be against going that direction... seems like there are now some religious opinions on the benefits of utility classes, but I'm not sold.

Reply:
Some of the grid mixins are obvious, but I'm not certain all of them are available as mixins. I vote on waiting on that until the documentation has improved.

Reply:
👍 , particularly for the blog. If we start using them heavily in the site pages, we can revisit this.